src/control/Control.Layers.js

```diff
@@ -85,11 +85,15 @@ export const Layers = Control.extend({
 		this._handlingClick = false;
 
 		for (const i in baseLayers) {

```

Comment:
Perhaps we can improve this by using `Object.entries()` instead? For example:

```js
for (const [name, layer] of Object.entries(baseLayers)) {
    this._addLayer(layer, name);
}
```

Reply:
We could make this switch if we were certain that `baseLayers` was not null or undefined. Is that guaranteed in this case? If not, we would have to add an additional null check, or stay with the `hasOwn` guard.